src/praisonai/praisonai/sandbox/daytona.py

```
@@ -0,0 +1,471 @@
+        self._client = None
+        self._is_running = False
+    
+    @property
+    def is_available(self) -> bool:
+        """Check if Daytona backend is available."""
+        try:
+            # Try to check for Daytona CLI or API availability
+            import subprocess
+            import shlex
+            # Check if daytona command exists
+            result = subprocess.run(["which", "daytona"], capture_output=True, text=True)
```

Ruff (0.15.10)

[error] Starting a process with a partial executable path

(S607)